src/ucp/core/ucp_ep.c

```diff
@@ -1416,6 +1416,69 @@ int ucp_ep_config_get_multi_lane_prio(const ucp_lane_index_t *lanes,
     return -1;
 }
 
+static char* ucp_ep_lane_add_desc(int prio, char *buf, size_t max,
+                                  char *desc, int *sep)
+{
+    if (prio != -1) {
+        snprintf(buf, max, "%*s%s#%d", *sep, "", desc, prio);
```

Comment:
minor: if I got your idea correctly, maybe better to use `"%.*s%s#%d", *sep, desc, prio`?